src/pages/home.js

```diff
@@ -0,0 +1,11 @@
+import { Navbar } from "../components/allComp"
+export const Home = () => {
+    return (
+        <div>
+            <Navbar />
```

Comment:
put navbar in app file before routes as it will be required in all components not only homepage

Reply:
Ok thanks for suggestion